### ISSUE TITLE
Check pipelines the API Readiness app targets

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#5254]
For .NET we are looking at the `` pipeline, but for Python we are looking at `` even though there is a ``. @dw511214992 could you explain to me the difference between the Python pipelines?

<img width="513" alt="image" src="https://user-images.githubusercontent.com/9868623/215205906-218f51e5-16a9-4414-ac72-65c4c83fe128.png">

**Comment by dw511214992** [comment]
for dataplane, python is using `azure-sdk-for-python`, `azure-sdk-for-python-track2` is for mgmt. I think we need to remove `-track2` for all pipelines, but .Net team said they will `-track2` until track2 .Net mgmt codegen is GA.

**Comment by maririos** [comment]
Thank you @dw511214992 . Is there documentation about this somewhere?
My concern is that the names are not clear and it will create confusion to our customers. Same as it happened to me

**Comment by dw511214992** [comment]
No, there is no any documents. But I think we can standarlize it because .Net track2 is GA now. (don't use -track2 for all languages)
